pkg/semrel/semrel_test.go

```diff
@@ -39,28 +39,30 @@ func TestApplyChange(t *testing.T) {
 		change                          *Change
 		expectedVersion                 string
 		allowInitialDevelopmentVersions bool
+		forceBumpPatchVersion           bool
 	}{
 		// No Previous releases
-		{"0.0.0", NoChange, "1.0.0", false},
-		{"0.0.0", PatchChange, "1.0.0", false},
-		{"0.0.0", MinorChange, "1.0.0", false},
-		{"0.0.0", MajorChange, "1.0.0", false},
+		{"0.0.0", NoChange, "1.0.0", false, false},
+		{"0.0.0", PatchChange, "1.0.0", false, false},
+		{"0.0.0", MinorChange, "1.0.0", false, false},
+		{"0.0.0", MajorChange, "1.0.0", false, false},
 
-		{"0.0.0", NoChange, "0.1.0", true},
-		{"0.0.0", PatchChange, "0.1.0", true},
-		{"0.0.0", MinorChange, "0.1.0", true},
-		{"0.0.0", MajorChange, "1.0.0", true},
+		{"0.0.0", NoChange, "0.1.0", true, false},
+		{"0.0.0", PatchChange, "0.1.0", true, false},
+		{"0.0.0", MinorChange, "0.1.0", true, false},
+		{"0.0.0", MajorChange, "1.0.0", true, false},
 
-		{"1.0.0", NoChange, "", false},
-		{"1.0.0", PatchChange, "1.0.1", false},
-		{"1.0.0", MinorChange, "1.1.0", false},
-		{"1.0.0", MajorChange, "2.0.0", false},
-		{"0.1.0", NoChange, "1.0.0", false},
-		{"0.1.0", NoChange, "", true},
+		{"1.0.0", NoChange, "", false, false},
+		{"1.0.0", PatchChange, "1.0.1", false, true},
```

Comment:
Here should be passed `NoChange`

Reply:
Okay understood, I just adapted the line in the PR